### PR TITLE
RAC-5253 add wss deb build

### PR DESCRIPTION
**Pull request by changev**
on-wss is necessary for on-web-ui

It should be released with on-web-ui

@panpan0000 @PengTian0 

Deb installation/uninstallation and service start/stop/status were tested successfully.

### CODE REVIEW

File: HWIMO-BUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2015, 2016, EMC, Inc.
```
Comment (PengTian0) [inline HWIMO-BUILD]: Please update it to 2017, DELLEMC

File: debian/changelog [hunk 2]
```diff
@@ -0,0 +1,5 @@
+on-wss (2.7.1) UNRELEASED; urgency=low
```
Comment (PengTian0) [inline debian/changelog]: Currently, the version of on-wss is 2.16.0.
Maybe, 2.16.0 is more reasonable.

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Source: on-wss
+Section: utilties
+Priority: optional
+Maintainer: Joe Heck <joseph.heck@emc.com>
+Build-Depends: debhelper (>= 8.0.0)
+Standards-Version: 3.9.4
+
+Package: on-wss
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, nodejs (>= 4.4.5-1nodesource1~), rabbitmq-server
```
Comment (PengTian0) [inline debian/control]: According to the README, on-wss depends on Mongodb and RabbitMQ .
Without MongoDB,  is it running  normally?

File: debian/on-wss.install [hunk 4]
```diff
@@ -0,0 +1,5 @@
+node_modules /var/renasar/on-tftp
+lib /var/renasar/on-tftp
+data /var/renasar/on-tftp
+index.js /var/renasar/on-tftp
+commitstring.txt /var/renasar/on-tftp
```
Comment (PengTian0) [inline debian/on-wss.install]: Why put all the files and folders under on-tftp?

Comment (changev) [reply debian/on-wss.install]: @PengTian0  That's a mistake~

**Pull request closed without merging**